dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -308,12 +295,18 @@ def main(args):
         t_p.append(throughput)
         print_key_value(key=f"{m}", value=f"{format_bytes(throughput)}/s")
         t_runs[idx] = float(format_bytes(throughput).split(" ")[0])
-    t_p = numpy.asarray(t_p)
-    times = numpy.asarray(times)
+    t_p = np.asarray(t_p)
+    times = np.asarray(times)
+    bandwidths_all = np.asarray(bandwidths_all)
     print_separator(separator="=")
     print_key_value(
         key="Throughput",
-        value=f"{format_bytes(t_p.mean())} +/- {format_bytes(t_p.std()) }",
+        value=f"{format_bytes(hmean(t_p))}/s +/- {format_bytes(hstd(t_p))}/s",
```

Comment:
I had a check to compare with how standard errors are propagated for reciprocals and these formulae are consistent with that approach. For this case `data_processed` is the same for every run and the times vary. So we can compute the mean throughput with `data_processed / np.mean(times)` and the standard deviation with `data_processed * np.std(times) / np.mean(times)**2`. But these are just special cases of the harmonic approach, so I'm happy.